### PR TITLE
Bk/feat/create properties

### PR DESCRIPTION
**Pull request by babslgam**
adds data and management command for adding properties and relations.

### CODE REVIEW

File: apis_ontology/management/commands/importproperties.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            prop.obj_class.clear()
+            for obj in p["obj"]:
+                prop.obj_class.add(ContentType.objects.get(model=obj.lower()))
+            prop.subj_class.clear()
+            for subj in p["subj"]:
+                prop.subj_class.add(ContentType.objects.get(model=subj.lower()))
+            if created:
+                print(f"Created {prop}")
+            else:
+                print(f"Updated {prop}")
```
Comment (koeaw) [inline apis_ontology/management/commands/importproperties.py]: Could you add a trailing newline at the end of the file and rename it to `import_properties.py`?

I know [PEP8](https://peps.python.org/pep-0008/#package-and-module-names) leaves room for interpretation regarding "readability" of file names, but find that using underscores as word separators helps (me) a lot with parsing them.

File: data/properties.json [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+	"subj": [
+			"Person",
+			"Organisation"
+	],
+	"obj": [
+			"Work",
+			"Expression"
+	]
+}
+]
```
Comment (koeaw) [inline data/properties.json]: This file should also end with a trailing newline.

**Review by koeaw — CHANGES_REQUESTED** [review]

File: apis_ontology/management/commands/importproperties.py (hunk 1 above)
Comment (babslgam) [reply apis_ontology/management/commands/importproperties.py]: updated it

File: data/properties.json (hunk 2 above)
Comment (babslgam) [reply data/properties.json]: also updated

**Comment by b1rger** [comment]
FYI: not sure if you're aware, but in apis, the properties and entities share the same `id` space, so creating a Property with `id=1` will clash with an Entity with `id=1` - in sicprod the `properties.json` only contains properties that already existed in the database (they were created at some point using the `construct_properties` method). We did it this way so we could rename properties.
To work around this you can use `Property.objects.get_or_create(name=p["name"], name_reverse=p["name_reverse"])` and ignore the `id`. Once https://github.com/acdh-oeaw/apis-core-rdf/issues/412 is fixed, it should be possible to easily create properties from the admin backend

**Comment by koeaw** [comment]
@babslgam Thanks for the update, could you rebase the existing commits to incorporate the formatting changes? No need to create extra commits for these. You should be able to force-push your own branches, which will update the PR.

(If that doesn't work, I'll have to check the repo settings to see if there's something blocking force pushes.)

**Comment by koeaw** [comment]
> To work around this you can use `Property.objects.get_or_create(name=p["name"], name_reverse=p["name_reverse"])` and ignore the `id`.

Thank you for catching this, @b1rger! I was wondering if it was necessary at all to assign IDs but for some reason assumed that @babslgam and you had had a conversation about the Sicprod management command. :see_no_evil: Oups.

**Comment by babslgam** [comment]
@b1rger no i'm not aware. i'm just desperately trying to make things work, because matej is asking me on a weekly basis about the status of the import for another month now. i just copied this method because i couldn't find the properties in the models anymore, where - as far as i know - they need to be in order to use the old way of creating properties. i would really apreciate if the instance could be updated by someone from apis group to a state where i just have to deal with the import of the data. (which excludes creation of properties)

**Comment by koeaw** [comment]
@babslgam Sorry, I wasn't aware they were missing from `models.py`, I can add them there or fix the PR, no worries.

**Pull request closed without merging**